src/Pager.tsx

```diff
@@ -35,6 +35,7 @@ type Props<T extends Route> = PagerCommonProps & {
       jumpTo: (key: string) => void;
     }
   ) => React.ReactNode;
+  gestureHandlerProps: object;
```

Comment:
Can you change it to `React.ComponentProps<typeof PanGestureHandler>`?

Reply:
Good point! Changed in both places.